### ISSUE TITLE
ycsb-0.9.0 missing cassandra2-binding

### ISSUE THREAD

**Issue by kruthar** [brianfrankcooper/YCSB#766]
Have seen this twice now. Is it worth trying to re-release ycsb-0.9.0? or should we just wait for ycsb-0.10.0. It seems sufficient to just point people at ycsb-0.8.0 for now.

**Comment by busbey** [comment]
we should just do 0.10.0. I'll start the RC process this weekend.

**Comment by busbey** [comment]
we should update the release notes on the 0.9.0 release though, since it calls out that binding as supported.

**Comment by busbey** [comment]
just to confirm, is this that the binding-specific tarball is missing from the release (I notice that some others are as well, accumulo for example) or is the issue that the needed binary bits aren't in the everything-ycsb-binary-tarball?

**Comment by kruthar** [comment]
The cassandra2-binding directory is not present when you untar the ycsb-0.9.0.tar.gz which I think is what you are dubbing the 'everything-ycsb-binary-tarball'.

I was under the impression that only the bindings that were tested for that release were provided in single binding packages. That seems to be the case this time. So, it makes since that there is not a ycsb-cassandra2-0.9.0.tar.gz because it was not tested this time around.

**Comment by busbey** [comment]
Anything in one of the "supported" categories is supposed to have a binding-specific tarball. If we just do those actually tested, folks will have to retest everything even when nonchanges should impact them to keep in "most blessed" status.

**Comment by busbey** [comment]
Does the YCSB binary tarball contain the cassandra2 binding in the current master branch?

**Comment by kruthar** [comment]
Do you mean is it present if I just try a `mvn package` on master right now? I am not sure, I'm failing the Accumulo tests because Zookeeper isn't starting. Is it supposed to kick off a full test of Accumulo and everything when you build on a laptop? (as opposed to builds done on travis).

**Comment by busbey** [comment]
Yeah, I think Accumulo tests get included by default in our build. Lemme check before cutting a branch for #772

**Comment by busbey** [comment]
I updated the release notes for 0.9.0 to call out this issue.

**Comment by busbey** [comment]
okay, looks like it's present in master:

```
Busbey-MBA:YCSB busbey$ tar tzf distribution/target/ycsb-0.10.0-SNAPSHOT.tar.gz | grep cassandra2-binding
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/README.md
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/cassandra-driver-core-3.0.0.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/guava-16.0.1.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/metrics-core-3.1.2.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/netty-buffer-4.0.33.Final.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/netty-codec-4.0.33.Final.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/netty-common-4.0.33.Final.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/netty-handler-4.0.33.Final.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/netty-transport-4.0.33.Final.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/slf4j-api-1.6.4.jar
ycsb-0.10.0-SNAPSHOT/cassandra2-binding/lib/cassandra2-binding-0.10.0-SNAPSHOT.jar
```

**Comment by busbey** [comment]
closed in #772
